src/Admin/Pool.php

```diff
@@ -291,6 +292,27 @@ public function getAdminByAdminCode($adminCode)
         return $admin;
     }
 
+    /**
+     * See if an admin with a certain admin code exists.
+     *
+     * @param string $adminCode
```

Comment:
Please, remove these docblocks which are not required now that you added type declarations.

Reply:
You are right! Sorry for that...